react-list.es6

```diff
@@ -72,6 +86,7 @@ module.exports = class ReactList extends Component {
   }
 
   componentDidMount() {
+    this.PASSIVE = supportsPassive() ? {passive: true} : false;
```

Comment:
Ah yes, I was thinking it would obviously crash on the server but the
try/catch or a process.browser check will clear that right up.

On Aug 25, 2016 11:36 AM, "Casey Foster" <notifications@github.com> wrote:

> In react-list.es6
> <https://github.com/orgsync/react-list/pull/131#discussion_r76278814>:
>
> > @@ -72,6 +86,7 @@ module.exports = class ReactList extends Component {
> >    }
> >
> >    componentDidMount() {
> > +    this.PASSIVE = supportsPassive() ? {passive: true} : false;
>
> Can't this just be a constant in the module scope?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/orgsync/react-list/pull/131/files/7cc25b30658d7f879ac100b766896cd857b12e21#r76278814>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABJFP50txIkKvSqJdKZXzb5Fvfpb3vyhks5qjcR1gaJpZM4JtOs9>
> .
>